dashboard/src/shared/AppRepository.ts

```diff
@@ -1,30 +1,29 @@
 import { axios } from "./Auth";
-import Config from "./Config";
 import { IAppRepository, IAppRepositoryList } from "./types";
 
 export class AppRepository {
-  public static async list() {
-    const { data } = await axios.get<IAppRepositoryList>(await AppRepository.getResourceLink());
+  public static async list(namespace: string) {
```

Comment:
is this namespace a kubeappsNamespace?

Reply:
yes, but I don't AppRepository should really know about that, it just cares about what namespace to request from in the API

Reply:
Agreed, thanks for the explanation.